microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Streamline your workflows with Edge Command Palette
```

Comment:
consider concrete specific title eg:
Run browser and DevTools commands via keyboard with Edge Command Palette